Xamarin.Essentials/DeviceDisplay/DeviceDisplay.shared.cs

```diff
@@ -4,121 +4,65 @@ namespace Xamarin.Essentials
+        public static DisplayInfo MainDisplayInfo => GetMainDisplayInfo();
 
-        public static event EventHandler<ScreenMetricsChangedEventArgs> ScreenMetricsChanged
+        static void SetCurrent(DisplayInfo metrics) =>
+            currentMetrics = new DisplayInfo(metrics.Width, metrics.Height, metrics.Density, metrics.Orientation, metrics.Rotation);
+
+        public static event EventHandler<MainDisplayInfoChangedEventArgs> MainDisplayInfoChanged
         {
             add
             {
-                var wasRunning = ScreenMetricsChangedInternal != null;
+                var wasRunning = MainDisplayInfoChangedInternal != null;
 
-                ScreenMetricsChangedInternal += value;
+                MainDisplayInfoChangedInternal += value;
 
-                if (!wasRunning && ScreenMetricsChangedInternal != null)
+                if (!wasRunning && MainDisplayInfoChangedInternal != null)
                 {
-                    SetCurrent(GetScreenMetrics());
+                    SetCurrent(GetMainDisplayInfo());
                     StartScreenMetricsListeners();
                 }
             }
 
             remove
             {
-                var wasRunning = ScreenMetricsChangedInternal != null;
+                var wasRunning = MainDisplayInfoChangedInternal != null;
 
-                ScreenMetricsChangedInternal -= value;
+                MainDisplayInfoChangedInternal -= value;
 
-                if (wasRunning && ScreenMetricsChangedInternal == null)
+                if (wasRunning && MainDisplayInfoChangedInternal == null)
                     StopScreenMetricsListeners();
             }
         }
 
-        static void OnScreenMetricsChanged(ScreenMetrics metrics)
-            => OnScreenMetricsChanged(new ScreenMetricsChangedEventArgs(metrics));
+        static void OnMainDisplayInfoChanged(DisplayInfo metrics)
+            => OnMainDisplayInfoChanged(new MainDisplayInfoChangedEventArgs(metrics));
 
-        static void OnScreenMetricsChanged(ScreenMetricsChangedEventArgs e)
+        static void OnMainDisplayInfoChanged(MainDisplayInfoChangedEventArgs e)
         {
-            if (!currentMetrics.Equals(e.Metrics))
+            if (!currentMetrics.Equals(e.MainDisplayInfo))
             {
-                SetCurrent(e.Metrics);
-                ScreenMetricsChangedInternal?.Invoke(null, e);
+                SetCurrent(e.MainDisplayInfo);
+                MainDisplayInfoChangedInternal?.Invoke(null, e);
             }
         }
     }
 
-    public class ScreenMetricsChangedEventArgs : EventArgs
-    {
-        public ScreenMetricsChangedEventArgs(ScreenMetrics metrics) =>
-            Metrics = metrics;
-
-        public ScreenMetrics Metrics { get; }
-    }
-
-    [Preserve(AllMembers = true)]
-    public readonly struct ScreenMetrics : IEquatable<ScreenMetrics>
+    public class MainDisplayInfoChangedEventArgs : EventArgs
```

Comment:
I don't think we should have the `Main` prefix here as we may want to use this for other screens.